src/test/java/dog/pawbook/model/managedentity/dog/DateOfBirthTest.java

```diff
@@ -15,7 +16,6 @@ public void constructor_null_throwsNullPointerException() {
 
     @Test
     public void constructor_invalidDateOfBirth_throwsIllegalArgumentException() {
-        String invalidDateOfBirth = "";
-        assertThrows(IllegalArgumentException.class, () -> new DateOfBirth(invalidDateOfBirth));
+        assertThrows(IllegalArgumentException.class, () -> new DateOfBirth(INVALID_DATEOFBIRTH_DESC));
     }
```

Comment:
should there be valid test cases as well for sanity checks?

Reply:
it would be good to test one or two problematic dates like 31-04-2000 etc, though the bulk of it should be tested against the parserutil. And if I'm not mistaken there are 2 constructors for this, one that accepts string and another that accepts LocalDate.

Reply:
hmm i didnt do this, just changed it to an available constant from the util class, but i can add some in!